.travis.yml

```diff
@@ -12,4 +12,4 @@ services:
 
 before_script: npm run start:memcached
 
-script: npm run test:e2e
+script: npm run lint && npm run test:e2e
```

Comment:
I think it's better to run two different scripts here for clarity.

Reply:
IIRC Travis only allows defining one build/test script, hence the need to concatenate commands. Is there any other way?

Reply:
https://github.com/bloqpriv/cloud-billing/blob/3b3f4133e448d85973dbdf0582c333efac01b8d1/.travis.yml#L19